MapboxDirections/Match/MBMatchOptions.swift

```diff
@@ -164,3 +173,8 @@ open class MatchOptions: DirectionsOptions {
         return (waypoints, routes)
     }
 }
+
+private protocol MatchOptionsDeprecations {
```

Comment:
This is [a neat trick](https://stackoverflow.com/a/45743766/4585461) for suppressing intentional implementation-related usage of deprecated class members.